components/automate-gateway/api/nodes/manager/manager.proto

```diff
@@ -169,62 +285,91 @@ service NodeManagerService {
 message ConnectResponse{}
 
 message Id {
+	// UUID for the nodemanager.
 	string id = 1;
 }
 message Ids {
+	// List of nodemanager UUIDs.
 	repeated Id ids = 1;
 }
 
 message Query {
+	// Filters to be applied to the query.
 	repeated chef.automate.domain.compliance.api.common.Filter filter_map = 25;
-
+	// Sort the results in ascending or descending order.
 	enum OrderType {
 		ASC = 0;
 		DESC = 1;
 	}
 	OrderType order = 21;
+	// Field to sort on.
 	string sort = 22;
+	// Page number of results to return.
 	int32 page = 23;
+	// Count of results that should be returned for each page.
 	int32 per_page = 24;
 }
 
 message Fields {
+	// List of available fields matching the requested criteria.
 	repeated string fields = 1;
 }
 
 message FieldQuery {
-		Query query = 20;
-		string field = 21;
-		string node_manager_id = 22;
+	// Query details (filters) to be applied to the results.
+	Query query = 20;
+	// Field to search on.
+	string field = 21;
+	// Nodemanager id for which the search is being made.
+	string node_manager_id = 22;
 }
 
 message Nodes {
-		repeated string nodes = 1;
-		int32 total = 20;
+	// List of node names matching the request.
+	repeated string nodes = 1;
+	// Total count of node names matching the request.
+	int32 total = 20;
 }
 
 message NodeQuery {
-		Query query = 20;
-		string node_manager_id = 21;
+	// Query details (filters) to be applied to the results.
+	Query query = 20;
+	// Nodemanager id for which the search is being made.
+	string node_manager_id = 21;
 }
 
 message NodeManager {
-		string id = 1;
-		string name = 2;
-		string type = 3;
-		string credential_id = 20; // aws or azure cred id
-		repeated CredentialsByTags instance_credentials = 21; // ssh, winrm creds
-		string status = 22;
-		string account_id = 24;
-		google.protobuf.Timestamp date_added = 25;
-		repeated chef.automate.domain.compliance.api.common.Kv credential_data = 26;
+	// UUID for the nodemanager.
+	string id = 1;
+	// User defined name for the nodemanager.
+	string name = 2;
+	// Type of nodemanager (aws-ec2, azure-vm, aws-api, azure-api, gcp).
+	string type = 3;
+	// UUID of credential containing the information to connect to aws, azure, or gcp.
+	string credential_id = 20;
+	// List of tag and credential uuid associations to make. These are ssh, winrm, and sudo creds used to access instances.
+	repeated CredentialsByTags instance_credentials = 21;
+	// Status of the nodemanager (reachable, unreachable).
+	string status = 22;
+	// Account id associated with the nodemanager.
```

Comment:
This is probably ok as a placeholder but do we know more about this? What account is this id? Is this something we get from aws or something?

Reply:
aws account id, azure tenant id, or gcp account id